packages/cli/src/commands/list.ts

```diff
@@ -42,14 +47,26 @@ const help = () => {
 
   ${chalk.dim('Examples:')}
 
-  ${chalk.gray('–')} List all deployments
+  ${chalk.gray('–')} List all deployments for the currently linked project
 
     ${chalk.cyan(`$ ${getPkgName()} ls`)}
 
-  ${chalk.gray('–')} List all deployments for the app ${chalk.dim('`my-app`')}
+  ${chalk.gray(
+    '–'
+  )} List all projects in the team of the currently linked project
```

Comment:
This sounds like its the same as `vc project ls`. I think we should keep it as a separate command instead of overloading the `vc ls` command

Reply:
I'm good with either, but if we remove `--all`, it may still be useful to include help text here saying to use `vc project ls` for this case.